pynitrokey/confconsts.py

```diff
@@ -47,3 +49,10 @@ class Verbosity(IntEnum):
 UDEV_URL = (
     "https://docs.nitrokey.com/nitrokey3/linux/firmware-update.html#troubleshooting"
 )
+
+
+logger = logging.getLogger(__name__)
+stream_handler = logging.StreamHandler()
+stream_handler.setLevel(VERBOSE)
+stream_handler.setFormatter(logging.Formatter(LOG_FORMAT_STDOUT))
+logger.addHandler(stream_handler)
```

Comment:
The default verbosity level should probably be higher than INFO to avoid printing to console.